### Analysis notebooks and data used in the manuscript: <br> ***"Gene-level alignment of single cell trajectories"*** <br>
[https://www.biorxiv.org/content/10.1101/2023.03.08.531713v3](https://doi.org/10.1101/2023.03.08.531713 ) (bioRxiv Preprint)

**Available datasets:**

1. Simulated trajectories using Gaussian Processes
2. Simulated trajectory perturbations using the mouse pancreas development dataset from CellRank (https://github.com/theislab/cellrank).
3. Negative control simulated dataset generated using the published script of Laidlaw et al (2023) at https://github.com/No2Ross/TrAGEDy.
4. Anndata objects of the PAM/LPS-treated murine bone marrow-dreived dendritic cell dataset of Shalek et al. (2014) compiled from the datafiles at https://github.com/shenorrLabTRDF/cellAlign.
5. Preprocessed and analysed Healthy/IPF datasets: <br>(original raw data of Adams et al. (2020) downloaded from GEO: GSE136831).
6. Preprocessed and analysed Pan fetal reference datasets: <br> (original raw data of Suo et al. (2022) downloaded from https://developmental.cellatlas.io/fetal-immune).
7. Preprocessed and analysed Artificial thymic organoid datasets: <br> (raw sequencing data available from ArrayExpress: E-MTAB-12720).
8. Additional data files: Human transcription factor list (Lambert et al. 2018) , cell-cycle gene list (Regev Lab), and relevant Gene Ontology pathway gene sets. 

Large data objects are available at: https://zenodo.org/records/11182400

All analysis notebooks use Genes2Genes v0.1.0 (https://github.com/Teichlab/Genes2Genes/releases/tag/v0.1.0)
